README.md

```diff
@@ -1,6 +1,28 @@
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.assertj/assertj-vavr/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.assertj/assertj-vavr)
+
 # assertj-vavr
-Assertions for Vavr - http://www.vavr.io/
+Assertions for [Vavr](http://www.vavr.io/)
+
+## Usage
+
+### Dependency in your project:
+- build.gradle:
+```groovy
```

Comment:
Why `groovy` here?

Reply:
build.gradle is usually created in groovy, right? Maybe it is superfluous

Reply:
No, it's all good :) I just misunderstood it.